src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+
+This is how I set up the environment with a sky dome, and a plane decorated with a couple of sad grey tones.
+
+```html
+<a-sky color="#333"></a-sky>
+<a-plane rotation="-90 0 0"
+         width="15" height="15" color="#777">
+</a-plane>
+```
+
+Length units in A-Frame are expressed in meters and angles are described in sexagesimal degrees (0º to 360º). Thus, the floor has a surface of _15m x 15m_, the workbench is _2cm_ thick and has a surface of _2m x 1m_, and the circuit board is _3mm thick_ and measures _5cm x 5cm_. Real measures are important for two reasons: they **improve the feeling of immersion** and provide **realistic physics**.
+
+When placing elements, take into account that they are placed according to their central point in the coordinate system of the parent element. If they are direct children of the `a-scene` element, then they are positioned according to the world coordinates system. In the case of the chip, **it is positioned relative to the central point of the circuit board** because the circuit board is the parent node of the chip.
+
+```html
+<a-box id="circuitboard" position="0 0.0115 0.4"
+       width="0.05" depth="0.05" height="0.003" color="#008000">
+  <a-box id="chip" position="0 0.002 0.005"
+         width="0.02" depth="0.02" height="0.001" color="#000">
+  </a-box>
+</a-box>
+```
+
+A-Frame provides a way of adding empty nodes to the scene or, in A-Frame nomenclature, an **entity**. [Entities](/docs/0.6.0/core/entity.html) are represented by the `a-entity` element and they have nothing associated with them: no geometry, model, or material. They do have implicit `rotation`, `position`, `scale` and `visible` attributes, which are automatically injected by the framework for all the elements within the scene.
+
+Adding the [`hand-controls`](/docs/0.6.0/components/hand-controls.html) attribute to an entity displays a hand model following the position and orientation of one of the tracked controllers of an HTC Vive or Oculus.
+
+```html
+<a-entity id="left-hand" hand-controls="left"></a-entity>
+<a-entity id="right-hand" hand-controls="right"></a-entity>
+```
+
+This is the resulting scene:
+
+[![Initial scene](/images/blog/hands-on/step1-initial-scene.png)](/images/blog/hands-on/step1-initial-scene.png)
+
+### One workspace to rule them all
+
+I was developing in front of a desk so for testing purposes I placed the elements as if they were resting on my real desktop only 5cm higher up. This allowed me to avoid my hands colliding  with my physical keyboard when I’m in VR mode. Adjusting the whole workspace at the same time is achieved by changing the `position` attribute of the `workspace` entity. Thanks to the real and virtual table heights matching, the demo has a great feeling of presence.
```

Comment:
`To recreate the simulated environment I placed the element as if they were resting on my real desk...`